sim/druid/insect_swarm.go

```diff
@@ -29,10 +29,10 @@ func (druid *Druid) registerInsectSwarmSpell() {
 			},
 		},
 
-		DamageMultiplier: 1 *
-			druid.TalentsBonuses.genesisMultiplier *
-			(1 + core.TernaryFloat64(druid.SetBonuses.balance_t7_2, 0.1, 0) +
-				core.TernaryFloat64(druid.HasMajorGlyph(proto.DruidMajorGlyph_GlyphOfInsectSwarm), 0.3, 0)),
+		DamageMultiplier: 1 +
```

Comment:
There is now a alternative `DamageMultiplierAdditive` on spells which can be used instead if you want. Since all of these are additive it wont make a difference, UNLESS there is some other effect which dynamically modifies either `spell.DamageMultiplier` or `spell.DamageMultiplierAdditive`.

Reply:
I'd rather remove that alternative multiplier alltogether, the only actual use case is for hunters, and they could simply handle it locally. More options = more ways of using them wrongly.

Reply:
Full agree that fewer options is better, but it can't be done for hunter locally without being a mess; hunters have some dynamic effects that need to stack additively with the other effects in `DamageMultiplierAdditive`.

I'm pretty sure warlocks are going to need it also, once I clean up their code.

But we can let this be for now. If you really want to remove it, it can be discussed in a separate PR.